Makefile

```diff
@@ -56,23 +64,33 @@ tag_latest:
 	docker tag $(NAME)/node-firefox:$(VERSION) $(NAME)/node-firefox:latest
 	docker tag $(NAME)/node-phantomjs:$(VERSION) $(NAME)/node-phantomjs:latest
 
-release: tag_latest
-	@if ! docker images $(NAME)/base | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/base version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	@if ! docker images $(NAME)/hub | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/hub version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	@if ! docker images $(NAME)/node-base | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/node-base version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	@if ! docker images $(NAME)/node-chrome | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/node-chrome version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	@if ! docker images $(NAME)/node-firefox | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/node-firefox version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	@if ! docker images $(NAME)/node-phantomjs | awk '{ print $$2 }' | grep -q -F $(VERSION); then echo "$(NAME)/node-phantomjs version $(VERSION) is not yet built. Please run 'make build'"; false; fi
-	docker push $(NAME)/base
-	docker push $(NAME)/hub
-	docker push $(NAME)/node-base
-	docker push $(NAME)/node-chrome
-	docker push $(NAME)/node-firefox
-	docker push $(NAME)/node-phantomjs
+check_release_version:
+	@if [[ ! $(VERSION) =~ [[:digit:]].[[:digit:]].[[:digit:]] ]]; then echo "'$(VERSION)' is not an acceptable revision for a release target."; false; fi
+
+release: check_release_version tag_latest deploy
 	@echo "*** Don't forget to create a tag. git tag v$(VERSION) && git push origin v$(VERSION)"
 
+check_dev_release_version:
+		@if [[ $(VERSION) =~ [[:digit:]].[[:digit:]].[[:digit:]] ]]; then echo "'$(VERSION)' is not an acceptable revision for a develop target."; false; fi
```

Comment:
I think docker is ambiguous or really has no say on the exact meaning of "latest" except that it's the version "pull" gives you if you don't specify a tag.   Since "latest" is a "moving" tag (i.e. you may get a different result tomorrow from pull) to me indicated it' can't really be "stable".  Certainly not "stable" enough to extend "From:"   

I agree let's just document our choices here in the README for users both doing pull as well as working on the repo.